### DISCUSSION TITLE
What bugs does the `<script> document.title = "{{ title }}" </script>` solve?

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#239] [Q&A — Template] [answered]
In [line 62 of `overrides/partials/meta.html`](https://github.com/ObsidianPublisher/template-netlify-vercel/blame/3e775bce5676d1141052c082c336715737cea535/overrides/partials/meta.html#L62) you have: 
```html
<script> document.title = "{{ title }}" </script>
```

[You say](https://github.com/ObsidianPublisher/template-netlify-vercel/issues/4#issuecomment-1786743688 "Why set `&lt;script&gt; document.title = &quot;{{ title }}&quot; &lt;/script&gt;` as default? · Issue #4 · ObsidianPublisher/template-netlify-vercel") that it is there as a hot fix for a bug? What bug is it?

**Comment by Mara-Li** [comment, marked as answer]
I forgot
